### PR TITLE
added cubicDE

### PR DESCRIPTION
**Pull request by TheBjoRedCraft**
added cubicDE

### CODE REVIEW

File: minecraft_servers/cubic/background.png
Comment (maaaathis) [inline minecraft_servers/cubic/background.png:1]: Compress this image

File: minecraft_servers/cubic/background@2x.png
Comment (maaaathis) [inline minecraft_servers/cubic/background@2x.png:1]: Compress this image

File: minecraft_servers/cubic/manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "server_name": "cubic",
+    "nice_name": "Cubic",
+    "direct_ip": "djhase.de",
+    "supported_languages": [
+      "de", "en"
+    ],
+    "social": {
+      "web": "https://djhase.de",
```
Comment (maaaathis) [inline minecraft_servers/cubic/manifest.json:9]: Website is not working: Error 405

File: minecraft_servers/cubic/manifest.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      "discord": "https://dc.djhase.de"
```
Comment (maaaathis) [inline minecraft_servers/cubic/manifest.json:10]: Custom Discord invites are reserved for partners only. Set a normal Discord invite link: https://discord.gg/[YOUR-INVITE]

File: minecraft_servers/cubic/manifest.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+    "server_name": "cubic",
+    "nice_name": "Cubic",
+    "direct_ip": "djhase.de",
+    "supported_languages": [
+      "de", "en"
```
Comment (maaaathis) [inline minecraft_servers/cubic/manifest.json:6]:
```suggestion
      "de", 
      "en"
```

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/cubic/background.png
Comment (TheBjoRedCraft) [reply minecraft_servers/cubic/background.png:1]: were
on the website that is given it is not working

File: minecraft_servers/cubic/background@2x.png
Comment (TheBjoRedCraft) [reply minecraft_servers/cubic/background@2x.png:1]: were
on the website that is given it is not working

File: minecraft_servers/cubic/manifest.json (hunk 1 above)
Comment (TheBjoRedCraft) [reply minecraft_servers/cubic/manifest.json]: the website is under maintenance

Comment (maaaathis) [reply minecraft_servers/cubic/manifest.json:9]: Then remove

**Comment by TheBjoRedCraft** [comment]
i have changed that from the bot

**Comment by TheBjoRedCraft** [comment]
but it does not work

**Review by DoJapHD — CHANGES_REQUESTED** [review]
Some requested changes are still open

**Comment by TheBjoRedCraft** [comment]
What do I have to edit??

File: minecraft_servers/cubic/manifest.json (hunk 1 above)
Comment (TheBjoRedCraft) [reply minecraft_servers/cubic/manifest.json:9]: Its fixed

Comment (maaaathis) [reply minecraft_servers/cubic/manifest.json:9]: Now it's a blank page. 
If you don't have a normal functional and informative website, then remove.

**Comment by DoJapHD** [comment]
https://github.com/LabyMod/server-media/pull/960#discussion_r1484146453
https://github.com/LabyMod/server-media/pull/960#discussion_r1484933163

**Comment by DoJapHD** [comment]
PR inactive

**Pull request closed without merging**